Application/EdFi.Ods.AdminApi/Infrastructure/Security/SecurityExtensions.cs

```diff
@@ -43,7 +42,7 @@ public static class SecurityExtensions
                 opt.AddEphemeralEncryptionKey();
                 opt.AddEphemeralSigningKey();
                 opt.DisableAccessTokenEncryption();
-                opt.SetIssuer(new Uri(authority));
+                opt.SetIssuer(new Uri(issuer));
```

Comment:
Please also remove line 23, and update any appSettings files that have a key for "Authority".